task_handler_layer/task_buoy/msg/ResizedImage.msg

```diff
@@ -0,0 +1,3 @@
+uint32 original_height
```

Comment:
Please make a wiki, it will take 5 minutes for you but half hour for me to figure out the right thing !